crates/zoon/src/style.rs

```diff
@@ -198,6 +288,36 @@ impl Drop for StyleGroupHandle {
 
 // ------ global_styles ------
 
+/// Set styles that are globally used in your application.
+/// Very convenient for customizing the design at one single place.
+/// # Example
+/// How to style every button in your app in few lanes. The [Button] element has
+/// the `button` class attached by default to it. So it is possible to overwrite
```

Comment:
Okay, is it semantically correct  if instead of :

`So it is possible to overwrite it` .

we have :

`StyleGroup let you defined the css properties for the selected class`

Reply:
Perhaps we don't need to add neither of them at all... You've already documented `StyleGroup` well.

Reply:
Okay, I did shorten the description.